docs/source/tutorials/GEDI/GEDI_L3_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6ddf6e47",
+   "metadata": {},
+   "source": [
+    "# GEDI L3 Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df05da5d",
+   "metadata": {},
+   "source": [
+    "## GEDI Instrument\n",
+    "\n",
+    "GEDI produces the first high resolution layer ranging observations of 3D structure of earth with precise measurements of forest canopy height, canopy vertical structure and surface elevation. GEDI is attached to the International Space Station (ISS) and collects data globally between 51.6° N and 51.6° S latitudes at the highest resolution and densest sampling of any light detection and ranging (lidar) instrument in orbit to date."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a0bc500",
+   "metadata": {},
+   "source": [
+    "## About GEDI L3 Data\n",
+    "\n",
+    "GEDI L3 data products are gridded by spatially interpolating Level 2 footprint estimates of canopy cover, canopy height,Leaf Area Index (LAI), vertical foliage profile and their uncertainties.\n",
+    "\n",
+    "Level 2 data contains terrain elevation, canopy height, RH metrics and Leaf Area Index (LAI). The raw waveforms are collected by GEDI system and processed to provide canopy height and profile metrics. These metrics provide easy to use and interpret information about the vertical distribution of canopy material. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4276c20c",
+   "metadata": {},
+   "source": [
+    "![Canopy](../../_static/canopy_footprint_level2.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "66209329",
+   "metadata": {},
+   "source": [
+    "Source: https://gedi.umd.edu/data/products/\n",
+    "\n",
+    "Sample of GEDI lidar waveform (Left). The light brown area under the curve represents the return energy from the canopy, while the dark brown area signifies the return from the underlying typography. The black line is cumulative return energey, starting from the bottom of the ground return (normalized to 0) to the top of canopy (normalized to 1). The diagram on the right shows the distribution of trees that produced the waveform."
```

Comment:
Same comment as the previous Level 2 comment

Reply:
Added `Figure` as prefix to make it more like caption.